src/test/kotlin/no/nav/syfo/motebehov/api/internad/v2/MotebehovVeilederADTilgangV2Test.kt

```diff
@@ -58,31 +56,27 @@ class MotebehovVeilederADTilgangV2Test {
     fun tearDown() {
         mockRestServiceServer.verify()
         mockRestServiceWithProxyServer.verify()
-        loggUtAlle(contextHolder)
     }
 
     @Test
     @Throws(ParseException::class)
     fun `veileder nektes tilgang`() {
-        loggInnVeilederADV2(contextHolder, VEILEDER_ID)
         mockSvarFraIstilgangskontroll(ARBEIDSTAKER_FNR, HttpStatus.FORBIDDEN)
-        assertThrows<ForbiddenException> { motebehovVeilederController.hentMotebehovListe(ARBEIDSTAKER_FNR) }
+        assertThrows<ForbiddenException> { kallHentMotebehovListe() }
```

Comment:
Dette er nok mer en personlig referanse, men jeg synes testene er mer lesbare med både `loggInn`-kallet  og kall til metoden man skal teste slik det var før.

Reply:
@kaja-nav Jeg gjorde det slik for å gjøre test-koden mindre bloaty. Du har likevel et poeng om at det er fint å vite at bruker har "logget seg inn" (med mock token) i testene. Jeg renamer "kall"-metodene (e.g. `kallHentMotebehovListe()` til `loggInnOgKallHentMotebehovListe()`), slik at det kommer tydeligere frem 👍